config/schedule.rb

```diff
@@ -17,6 +17,10 @@
   rake "order_details:remove_merge_orders"
 end
 
+every :day, at: "12:30am", roles: [:db] do
```

Comment:
Let's make sure to double check when we release this that the time zone is taken into account.